components/admin_console/team_channel_settings/line_switch.tsx

```diff
@@ -22,7 +22,7 @@ type Props = {
 const LineSwitch = (props: Props): JSX.Element => {
     const {title, subTitle, singleLine, toggled, onToggle, children, offText, onText, disabled, last, id} = props;
     return (<div>
-        <div className='line-switch d-flex flex-sm-column flex-md-row align-items-sm-start align-items-center justify-content-md-between'>
+        <div className='line-switch d-flex flex-sm-column flex-md-row align-items-sm-start align-items-center justify-content-between'>
```

Comment:
i am not sure why we do this approach and not just create a css file for this component.

Reply:
@M-ZubairAhmed I find these helper classes on flexboxes actually helpful, 
I tend to use them in code.
If we are going away from this approach I can create a new class to include all relevant CSS.

Reply:
i am not sure whats the preferred approach here is. so i am going to approve it nevertheless. defering to @hmhealey for comments on that.

Reply:
I've already made the change you proposed BTW :)